jobs/loggregator_trafficcontroller/templates/drain.erb

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+# vim: set ft=sh
```

Comment:
Should we remove this editor-specific config? It seems like that would have difficulty being accepted upstream, at least.

Reply:
Copy paste from another drain script, removed.